cmake/projects/CppJwt/hunter.cmake

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2016-2019, Ruslan Baratov
+# Copyright (c) 2019 Richard Hodges
+# Copyright (c) 2019 Quoine Corporation
+# All rights reserved.
+
+# !!! DO NOT PLACE HEADER GUARDS HERE !!!
+
+include(hunter_add_version)
+include(hunter_cacheable)
+include(hunter_download)
+include(hunter_pick_scheme)
+include(hunter_configuration_types)
+
+hunter_add_version(
+    PACKAGE_NAME
+    CppJwt
+    VERSION
+    1.2.0-hunter0
+    URL
+    "https://github.com/QuoineFinancial/cpp-jwt/archive/v1.2.0-hunter0.tar.gz"
+    SHA1
+    0225e8533f9f5bd2f14d009ed1c8f29ced143092
+)
+
+hunter_configuration_types(CppJwt CONFIGURATION_TYPES Release)
```

Comment:
Why we have to force `Release`?

Reply:
It's a header-only library. There's no reason to release more than one version. Can remove if you like.

Reply:
If this library is header-only the overhead of using Release+Debug instead of Release is minimal. Please remove the unnecessary `hunter_configuration_types(CppJwt CONFIGURATION_TYPES Release)`.